config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
```

Comment:
- [ ] Whenever I run the rails s it is redirecting to an empty page instead of  http://127.0.0.1:3000/users/sign_in


![image](https://user-images.githubusercontent.com/75126481/217685115-f907179a-2e13-4876-87ce-38afcdcb1707.png)